services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> CinemaHall:

```

Comment:
The return type hint for `get_cinema_halls` should be `QuerySet[CinemaHall]` instead of `CinemaHall`, as the function returns a QuerySet containing multiple `CinemaHall` instances.